tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_NC_RTL.py

```diff
@@ -7,43 +7,196 @@
+        sim_page.tap_sim_security()
+        sim_page.enable_sim_pin()
+        self.take_screenshot('sim_manager-pin_page')
+        settings.return_to_prev_menu(sim_page.security_screen_element, sim_page.sim_pin_screen_element)
+        self.take_screenshot('sim_manager-security')
+        settings.return_to_prev_menu(sim_page.screen_element, sim_page.security_screen_element)
+        settings.return_to_prev_menu(settings.screen_element, sim_page.screen_element)
 
-        settings.open_call()
-        self.take_screenshot('call')
-        settings.return_to_prev_menu(settings.screen_element)
+        ################### Call Settings and its subpages ######################
+        cs_page = settings.open_call()
+        self.take_screenshot('cs-selectSIM')
+        cs_page.tap_sim_1()
+        self.take_screenshot('callsettings')
+        cs_page.tap_voicemail()
+        self.take_screenshot('voicemail')
+        settings.return_to_prev_menu(cs_page.cs_screen_element, cs_page.vm_screen_element)
 
-        settings.open_message()
+        cs_page.tap_caller_id_selection()
+        self.take_screenshot('callerID')
+        cs_page.confirm_caller_id_selection()
+
+        cs_page.tap_fixed_dialing()
+        self.take_screenshot('fixeddialing')
+        cs_page.tap_auth_numbers()
+        self.take_screenshot('authNumbers')
+        settings.return_to_prev_menu(cs_page.fd_screen_element, cs_page.auth_screen_element)
+        cs_page.enable_fixed_dialing()
+        self.take_screenshot('enableFDN')
+        settings.return_to_prev_menu(cs_page.fd_screen_element, cs_page.fd_simpin_screen_element)
+        settings.return_to_prev_menu(cs_page.cs_screen_element, cs_page.fd_screen_element)
+
+        cs_page.tap_call_forwarding()
+        self.take_screenshot('callforward_init')
+        cs_page.wait_until_cf_info_received()
+        self.take_screenshot('callforward')
+        cs_page.tap_always_forward()
+        self.take_screenshot('alwaysFwd')
+        settings.return_to_prev_menu(cs_page.cf_screen_element, cs_page.cf_always_screen_element)
+        cs_page.tap_forward_when_busy()
+        self.take_screenshot('busyFwd')
+        settings.return_to_prev_menu(cs_page.cf_screen_element, cs_page.cf_busy_screen_element)
+        cs_page.tap_forward_unanswered()
+        self.take_screenshot('unansweredFwd')
+        settings.return_to_prev_menu(cs_page.cf_screen_element, cs_page.cf_unans_screen_element)
+        cs_page.tap_forward_unreachable()
+        self.take_screenshot('unreachFwd')
+        settings.return_to_prev_menu(cs_page.cf_screen_element, cs_page.cf_unreach_screen_element)
+        settings.return_to_prev_menu(cs_page.cs_screen_element, cs_page.cf_screen_element)
+
+        cs_page.tap_call_barring()
+        self.take_screenshot('callBarring-init')
+        cs_page.wait_until_cb_info_received()
+        self.take_screenshot('callBarring')
+        cs_page.tap_change_passcode()
+        self.take_screenshot('callBarring-passcode')
```

Comment:
actually, in this dialog, its the 4 square without the caret, so that was good, i think in other cases, it opens up with the text area selected.